### DISCUSSION TITLE
Comments not Working

### DISCUSSION THREAD

**Discussion by Prottoy2938** [remarkjs/remark#562] [Q&A] [answered]
I'm using [react-markdown](https://github.com/remarkjs/react-markdown) package to render markdown.

So far everything working as expected, except for the comments.

When doing comments like this: `<!-- write markdown here -->`, it gets rendered on the screen like this:

![image](https://i.ibb.co/CWz12Wc/Create-New-Survey-Light-Bulb.png)

My component code: 

```jsx
import React from "react";
import { Props } from "./add-content.model";
import ReactMarkdown from "react-markdown";
import gfm from "remark-gfm";
import "github-markdown-css";


const AddContent: React.FC<Props> = (props: Props) => {
  const {content} = props

  return (
    <ReactMarkdown plugins={[gfm]}>{content}</ReactMarkdown>
  );
};

export default AddContent;


```

**Comment by wooorm** [comment, marked as answer]
Hi there! See the [`skipHtml`](https://github.com/remarkjs/react-markdown#props) option.
